assets/aml-benchmark/components/batch-benchmark-inference/spec.yaml

```diff
@@ -61,26 +61,14 @@ inputs:
     type: string
     optional: False
     description: Connections name for the endpoint.
-  metadata_key:
-    type: string
-    optional: True
-    description: The metadata key that in the request payload that contains label columns. If not provided, the `request_metadata` will be used in the batch benchmark score output to retrieve ground truth.
   label_key:
```

Comment:
nit: Can we rename this label_column_name?

Reply:
changed